### ISSUE TITLE
Permission denied errors when running v-restore-user

### ISSUE THREAD

**Issue by ifaist0s** [hestiacp/hestiacp#793]
### In a few words, please describe the issue that you're experiencing:
The following permission errors occur when restoring a user (Hestia > Hestia) via command line `v-restore-user`. The user being restored, has **password protected Awstats**.
```
-- WEB --
2020-05-02 16:43:13 [FQDN]
tee: /home/[USERNAME]/web/[FQDN]/stats/.htaccess: Permission denied AuthUserFile /home/[USERNAME]/web/[FQDN]/stats/.htpasswd
tee: /home/[USERNAME]/web/[FQDN]/stats/.htaccess: Permission denied AuthName "Web Statistics"
tee: /home/[USERNAME]/web/[FQDN]/stats/.htaccess: Permission denied AuthType Basic
tee: /home/[USERNAME]/web/[FQDN]/stats/.htaccess: Permission denied Require valid-user
tee: /home/[USERNAME]/web/[FQDN]/stats/.htpasswd: Permission denied [STATS_USER]:[STATS_CRYPT].
```
### What steps did you take when the issue occured? 
1. Place backup file in `/backup` folder
2. Change owner to admin:admin
3. Run `v-restore user [USER] [BACKUP]`

### Expected behavior:
The backup should complete without any errors.

### Does this issue always occur, or is it intermittent?
It occurs 100% of the times I tried, **when the user being restored does not already exist**.

### Operating system distribution and release:
Ubuntu 18.04.4 LTS (where the backup file was created at)
Ubuntu 18.04.4 LTS (where the backup is being restored to)

### Which version of Hestia Control Panel is currently installed?
1.1.1 (where the backup is being restored to)
1.0.6 (where the backup file was created at)

### Which branch are you using?
Release

### When did this issue occur? After a clean installation, or after an upgrade?
Clean installation of Hestia 1.1.1

The backup file was created on a server with Hestia 1.0.6, which might have been updated from an earlier Hestia version

### Which of the following web server configurations are you using?
- Apache + Nginx (default) + Multi-PHP (–nginx yes --apache yes --phpfpm no --multiphp yes)

### Related issues or forum threads:
I have described the issue in more detail here https://forum.hestiacp.com/t/issue-with-restore/765

### Additional notes:
I think that moving the code block `# Set folder permissions` at lines [226](https://github.com/hestiacp/hestiacp/blob/6d3c99e7a16b85a71a9f2f6f11c3b7b8810c4b9c/func/rebuild.sh#L226)-235 of [rebuild.sh](https://github.com/hestiacp/hestiacp/blob/master/func/rebuild.sh) at the end of the `rebuild_web_domain_conf()` function, i.e. below the block of `# Checking web server include` at lines [369](https://github.com/hestiacp/hestiacp/blob/6d3c99e7a16b85a71a9f2f6f11c3b7b8810c4b9c/func/rebuild.sh#L369)-383, would solve the problem.

**Comment by ifaist0s** [comment]
Solved this by [rearranging the setting of folder permissions](https://github.com/ifaist0s/hestiacp/blob/rebuild_web_domain-permissions/func/rebuild.sh) inside the `rebuild_web_domain_conf()` function.

One thing to note here... Running the restore I get this output:
```
-- WEB --
2020-05-03 10:38:36 dimakopouloi.gr
AuthUserFile /home/[USER]/web/[FQDN]/stats/.htpasswd
AuthName "Web Statistics"
AuthType Basic
Require valid-user
[STATS_USER]:[STATS_CRYPT]
```
Is this output really necessary? Especially the `[STATS_USER]:[STATS_CRYPT]` part? (lines 309-321 in `rebuild.sh`) Although to be fair, the existence of tee (and thus the output on the screen) was the reason to spot this issue.

Anyway, I'm making this pull request for the devs to consider. There might be implications by moving the setting of folder permissions at the end of the `rebuild_web_domain_conf()` function.

(bah, I opened a pull request in my own branch before opening the correct one >.< Sorry for the confusion)
